src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -102,12 +209,10 @@ public function iAddIt(): void
      */
     public function theProductShouldAppearInTheShop(string $productName): void
     {
-        $this->client->index();
+        $response = $this->client->index();
 
         Assert::true(
-            $this
-                ->responseChecker
-                ->hasItemWithTranslation($this->client->getLastResponse(),'en_US', 'name', $productName)
+            $this->responseChecker->hasItemWithTranslation($response,'en_US', 'name', $productName)
```

Comment:
```suggestion
            $this->responseChecker->hasItemWithTranslation($this->client->index(), 'en_US', 'name', $productName)
```

Reply:
I'm not sure if it wasn't my recommendation, because the line was too long. Maybe we can leave it as it is?